clients/web/src/views/layout/HeaderView.js

```diff
@@ -40,7 +40,7 @@ var LayoutHeaderView = View.extend({
         this.$el.html(LayoutHeaderTemplate({
             brandName: this.brandName
         }));
-
+        this.$('.g-header-wrapper').css('background-color', this.bannerColor);
```

Comment:
In my opinion, in order of priority, I think CSS information ought to be set:
1. In a stylesheeet
2. On an explicit `style="..."` attribute in the HTML
3. At runtime in Javascript

While 1 is not possible to do dynamically, we could do 2 by passing the value into the Pug template.

This isn't ideal, but IMO, it's a little better at maintaining the (content / style / behavior) distinction of (HTML / CSS / Javascript), and may make it slightly easier for a naive developer to notice where the header bar color is set dynamically. However, maybe other @girder/developers feel differently?